pep-0636.rst

```diff
@@ -359,7 +359,9 @@ this alternative definition::
 
     class Click:
         __match_args__ = ("position", "button")
-        def __init__(self, position, button):
+        def __init__(self, position_arg, button_arg):
```

Comment:
This naming for args is pretty stilted. If you want the args to differ from the attributes, I propose shortening them:
```suggestion
        def __init__(self, pos, btn):
```